envs/alfajores/migration-config/migration.env

```diff
@@ -1,2 +1 @@
 MIGRATION_BLOCK_NUMBER=26384000
-MIGRATION_BLOCK_TIME=1727339320
```

Comment:
Do we need to hardcode those for the testnets?

Reply:
We hardcoded those for the testnets [here](https://github.com/celo-org/optimism/pull/342/files#diff-e9ab083305d85cb9fea217fc18988c2748f3918b328559125eba8db4e33e6da4R313). It meant we could get rid of one flag for the migration script.

Reply:
Ah nice, I missed that!